docs/README.iOS.md

```diff
@@ -126,6 +131,34 @@ Build a specific group of add-ons:
 make -j$(getconf _NPROCESSORS_ONLN) -C tools/depends/target/binary-addons ADDONS="pvr.*"
 ```
 
+## 5.2. Xcode project building
+
+Binary addons will be built as a dependency in the Xcode project. You can choose the addons you wish to build during the Xcode project generation step
+
+Generate Xcode project to build specific add-ons:
+```
+make -C tools/depends/target/cmakebuildsys CMAKE_EXTRA_ARGUMENTS="-DENABLE_XCODE_ADDONBUILD=ON -DADDONS_TO_BUILD='audioencoder.flac pvr.vdr.vnsi audiodecoder.snesapu'"
+```
+
+Generate Xcode project to build a specific group of add-ons:
+```
+make -C tools/depends/target/cmakebuildsys CMAKE_EXTRA_ARGUMENTS="-DENABLE_XCODE_ADDONBUILD=ON -DADDONS_TO_BUILD='pvr.*'"
+```
+
+**TIP:** When using addontype.* for -DADDONS_TO_BUILD argument, you cannot have multiple addon types. ie -DADDONS_TO_BUILD='game.libretro.* peripheral.*' is not valid. You will need to individually list the addons as per the first example for specfiic addon building.
+
+Generate Xcode project to build all add-ons automatically:
+```
+make -C tools/depends/target/cmakebuildsys CMAKE_EXTRA_ARGUMENTS="-DENABLE_XCODE_ADDONBUILD=ON"
+```
+
+**TIP:** If you wish to not automatically build addons added to your xcode project, omit `-DENABLE_XCODE_ADDONBUILD=ON`. The target will be added to the project, but the dependency will not be set to automatically build
+**TIP:** Binary add-ons added to the generated Xcode project can be built independently of the Kodi app by selecting the scheme/target `binary-addons` in the Xcode project.
+You can also build the binary-addons target via xcodebuild. This will not build the Kodi App, but will build any/all binary addons added for the project Generation.
+```
+xcodebuild -config "Debug" -target binary-addons
```

Comment:
Maybe add here as **\`\`\`sh** or **\`\`\`bash** to color some parts on them

```sh
xcodebuild -config "Debug" -target binary-addons
```

Reply:
Have added. Doesnt really add much benefit in my eyes, but doesnt do any harm either.